Datadog/E2ETests/RUM/RUMMonitorE2ETests.swift

```diff
@@ -0,0 +1,679 @@
+
+        measure(resourceName: DD.PerfSpanName.fromCurrentMethodName()) {
+            rum.stopView(key: viewKey, attributes: [:])
+        }
+
+        rum.stopResourceLoading(resourceKey: resourceKey, statusCode: (200...500).randomElement()!, kind: .other)
+    }
+
+    /// - data monitor:
+    /// ```rum
+    /// $monitor_id = rum_rummonitor_stop_view_with_pending_action
+    /// $monitor_name = "[RUM] [iOS] Nightly - rum_rummonitor_stop_view_with_pending_action: number of views is below expected value"
+    /// $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_stop_view_with_pending_action @type:view @view.name:rumView* @view.url_path:datadog\/rum*\").rollup(\"count\").last(\"1d\") < 1"
+    /// ```
+    func test_rum_rummonitor_stop_view_with_pending_action() {
+        let viewKey = String.mockRandom()
+        let viewName = String.mockRandom()
+        let actionName = String.mockRandom()
+        let actionType = actionTypePool.randomElement()!
+
+        rum.startView(key: viewKey, name: viewName, attributes: DD.logAttributes())
+        rum.startUserAction(type: actionType, name: actionName, attributes: DD.logAttributes())
+
+        measure(resourceName: DD.PerfSpanName.fromCurrentMethodName()) {
+            rum.stopView(key: viewKey, attributes: [:])
+        }
+
+        rum.stopUserAction(type: actionType, name: actionName, attributes: [:])
+    }
+
+    // TODO: RUMM-1378 multiple monitors below can't be parsed
+
+    /// - data monitor:
+    /// ```rum
+    /// $monitor_id = rum_rummonitor_add_timing
+    /// $monitor_name = "[RUM] [iOS] Nightly - rum_rummonitor_add_timing: number of views is below expected value"
+    /// $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_add_timing @type:view @view.name:rumView* @view.url_path:datadog\/rum*\").rollup(\"count\").last(\"1d\") < 1"
+    /// ```
+    /// - data monitor:
+    /// ```rum
+    /// $monitor_id = rum_rummonitor_add_timing_upper_bound
+    /// $monitor_name = "[RUM] [iOS] Nightly - rum_rummonitor_add_timing_upper_bound: timing is below expected value"
+    /// $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_add_timing @type:view @view.url_path:datadog\/rum*\").rollup(\"avg\", \"@view.custom_timings.time_event\").last(\"1d\") < 200000000"
+    /// ```
+    /// - data monitor:
+    /// ```rum
+    /// $monitor_id = rum_rummonitor_add_timing_lower_bound
+    /// $monitor_name = "[RUM] [iOS] Nightly - rum_rummonitor_add_timing_lower_bound: timing is above expected value"
+    /// $monitor_query = "rum(\"service:com.datadog.android.nightly @context.test_method_name:rum_rummonitor_add_timing_lower_bound @type:view @view.url_path:datadog\/rum*\").rollup(\"avg\", \"@view.custom_timings.time_event\").last(\"1d\") > 700000000"
+    /// ```
```

Comment:
Android has 3 RUM monitors from this test case but it seems our tool doesn't support multiple RUM monitors, does it somehow?